samples/java-application-with-reflection/settings.gradle

```diff
@@ -45,6 +45,7 @@ pluginManagement {
     }
     repositories {
         mavenCentral()
+        mavenLocal()
```

Comment:
Will need to remove this from draft PR

Reply:
Ah yes, I've kept this to make it easier to test locally on multiple machines, I will remove all changes related to local testing